index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="icon" href="./images/icons-chatbot.png">
+  <link rel="stylesheet" href="style.css">
+  <title>Chat Bot</title>
+</head>
+
+<body>
+
+
+
```

Comment:
@raaj-oo7 Please make a habit of formatting your code.
This blank lines un-necessarily should not be committed